src/main/frontend/handler/page.cljs

```diff
@@ -877,3 +878,14 @@
   (if-let [file-path (and (util/electron?) (page-util/get-page-file-path))]
     (js/window.apis.showItemInFolder file-path)
     (notification/show! "No file found" :warning)))
+
+(defn copy-page-url [] 
+ (when-let [page-name (or
+                        (state/get-current-page)
+                        (state/get-current-whiteboard))]
+(let [page-name (util/page-name-sanity-lc page-name)
+          repo (state/sub :git/current-repo)
+          page (db/entity repo [:block/name page-name])
+          page-original-name (:block/original-name page)]
```

Comment:
The protocol page is flexible enough to take block/name or block/original-name so we can just use the page-name above